### PR TITLE
Intro to stacks

### PR DESCRIPTION
**Pull request by IsisJ**

### CODE REVIEW

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 1]
```diff
@@ -7,25 +7,19 @@
...
 		for(int i = 0; i< randomDoublesSize; i++) {
 			Random random = new Random();
 			Double ranNum = random.nextDouble()*100;
 			randomDoubles.push(ranNum);
 		}
 		
-		randomDoubles.peek();
+		//randomDoubles.peek();
 		
 		//3. Ask the user to enter in two numbers between 0 and 100, inclusive.
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: delete commented instructions 😃

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 2]
```diff
@@ -38,7 +32,7 @@ public static void main(String[] args) {
 		//   between the two numbers entered by the user, print it to the screen.
 		
 		
-		System.out.println("Popping elements between " +userInput1+" and "+userInput2+" off stack:");
+		
 		
 		//maybe find a way to put the between numbers in the right order...if that makes sense?
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: is this a note you left for yourself? if so please delete ✌️

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 3]
```diff
@@ -49,9 +43,21 @@ public static void main(String[] args) {
 		
 		int userNumInput1 = Integer.parseInt(userInput1);
 		int userNumInput2 = Integer.parseInt(userInput2);
-		for(int i =0; i < userNumInput2; i++) {
-			if(i > userNumInput1) {
-				System.out.println(randomDoubles.pop());
+		
+		if(userNumInput1-userNumInput2 < 0) {
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: There's an easier way to do this where you don't have to use subtraction...

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 4]
```diff
@@ -49,9 +43,21 @@ public static void main(String[] args) {
...
+			smallerNum = userNumInput1;
+			biggestNum = userNumInput2;
+		}else {
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java:31]: good use of else!

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 5]
```diff
@@ -7,21 +7,59 @@
...
+		int randomDoublesSize = 100;
 		
-		//2. Use a loop to push 100 random doubles between 0 and 100 to the Stack.
+		for(int i = 0; i< randomDoublesSize; i++) {
+			Random random = new Random();
+			Double ranNum = random.nextDouble()*100;
+			randomDoubles.push(ranNum);
+		}
+		
+		//randomDoubles.peek();
```
Comment (DCamaron1) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: You can delete any unwanted comments 😃

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 6]
```diff
@@ -49,9 +43,21 @@ public static void main(String[] args) {
...
+			smallerNum = userNumInput1;
+			biggestNum = userNumInput2;
+		}else {
+			smallerNum = userNumInput2;
+			biggestNum = userNumInput1;
+		}
+		
+		System.out.println("Popping elements between " +smallerNum+" and "+biggestNum+" off the stack:");
+		
+		while(!randomDoubles.empty()) {
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java:38]: cool way to use while!

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 7]
```diff
@@ -49,9 +43,21 @@ public static void main(String[] args) {
...
+			double beingPopped = randomDoubles.pop();
+			if(beingPopped > smallerNum && beingPopped < biggestNum) {
+				System.out.println(beingPopped);
 			}
 		}
 		//   Popping elements off stack...
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: delete left over instructions 👧

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 8]
```diff
@@ -7,21 +7,59 @@
...
+			smallerNum = userNumInput1;
+			biggestNum = userNumInput2;
+		}else {
+			smallerNum = userNumInput2;
+			biggestNum = userNumInput1;
+		}
+		
+		System.out.println("Popping elements between " +smallerNum+" and "+biggestNum+" off the stack:");
+		
+		while(!randomDoubles.empty()) {
```
Comment (DCamaron1) [inline src/_03_IntroToStacks/_01_IntroToStack.java:38]: Great use of new methods

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 9]
```diff
@@ -60,14 +41,5 @@ public static void main(String[] args) {
...
 		}
-		//   Popping elements off stack...
-		//   Elements between 65 and 75:
-		//   66.66876846
-		//   74.51651681
-		//   70.05110654
-		//   69.21350456
-		//   71.54506465
-		//   66.47984807
-		//   74.12121224
```
Comment (epuzio) [inline src/_03_IntroToStacks/_01_IntroToStack.java:33]: lgtm! 👍 👍 👍

**Review by mwoguri — APPROVED** [review]
LGTM!  Please merge.

**Pull request merged**